ext/CMakeLists.txt

```diff
@@ -68,13 +68,15 @@ endif(TRACY_ENABLE)
 CPMAddPackage(
     NAME fmt
     GITHUB_REPOSITORY fmtlib/fmt
-    GIT_TAG 10.1.1
+    GIT_TAG 10.2.1 #e69e5f977d458f2650bb346dadf2ad30c5320281
+    VERSION 10.2.1 #11.0.2 - causes errors
 ) # defines: fmt::fmt
 
 CPMAddPackage(
     NAME spdlog
     GITHUB_REPOSITORY gabime/spdlog
-    GIT_TAG v1.12.0
+    GIT_TAG v1.14.1 #27cb4c76708608465c413f6d0e6b8d99a4d84302
+    VERSION 1.14.1
```

Comment:
From the docs:
```
The origin may be specified by a GIT_REPOSITORY, but other sources, such as direct URLs, are [also supported](https://cmake.org/cmake/help/v3.11/module/ExternalProject.html#external-project-definition). If GIT_TAG hasn't been explicitly specified it defaults to v(VERSION), a common convention for git projects. On the other hand, if VERSION hasn't been explicitly specified, CPM can automatically identify the version from the git tag in some common cases. GIT_TAG can also be set to a specific commit or a branch name such as master, however this isn't recommended, as such packages will only be updated when the cache is cleared.
```

Is it redundant to specify both GIT_TAG and VERSION? I'd rather have just the tag and the comment with the commit hash, if we're not using just a specific hash